### ISSUE TITLE
Continuous Deployment of Bridge

### ISSUE THREAD

**Issue by guineveresaenger** [pulumi/ci-mgmt#581]
We should roll out bridge updates as they happen, not as they come up as part of other upgrades.

The most basic implementation here can be a requirement to manually fire off the "Upgrade Bridge, Ecosystem Providers" Job as part of a release.

We can consider a nightly run as well, or implement some sort of automatic trigger that looks for a new version and fires off if one is present. Note that we should consider both bridge and bridge/pf versions.

**Comment by t0yv0** [comment]
Similar to items in https://github.com/pulumi/home/issues/2545 yes?

**Comment by t0yv0** [comment]
This was claimed complete https://github.com/pulumi/home/issues/2589 but  must've regressed.

**Comment by guineveresaenger** [comment]
I don't think this was ever fully implemented. All the pieces in #2589 are there.

What's missing is the automatic triggering of a bridge version update rollout. It's manual and AFAICT it always has been.
